media-libs/libopenshot-audio/libopenshot_audio-0.1.6.recipe

```diff
@@ -0,0 +1,106 @@
+SUMMARY="Video editing library used by OpenShot"
+DESCRIPTION="OpenShot Audio Library (libopenshot-audio) is a program that \
+allows the high-quality editing and playback of audio, and is based on the \
+amazing JUCE library."
+HOMEPAGE="http://www.openshot.org"
+COPYRIGHT="2015 ROLI Ltd.
+	2008-2016 OpenShot Studios, LLC"
+LICENSE="GNU GPL v3
+	GNU GPL v2
+	GNU AGPL v3
+	ISC"
+REVISION="1"
+SOURCE_URI="https://github.com/OpenShot/libopenshot-audio/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="090ed62a59f35642081f6da58677e5f8f316ed2062cb47cf0617a71678b890c3"
+SOURCE_FILENAME="libopenshot-audio-$portVersion.tar.gz"
+SOURCE_DIR="libopenshot-audio-$portVersion"
+PATCHES="libopenshot_audio-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
Suggestion:
```diff
-SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
```
(I think there's an implicit rule which says that we don't need to mention broken architectures in `SECONDARY_ARCHITECTURES`.)

(Same suggestion for the other recipes.)